apps/www/components/ui/navigation-menu.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu"
+import { cva, type VariantProps } from "class-variance-authority"
+import { ChevronDown } from "lucide-react"
+
+import { cn } from "@/lib/utils"
+
+export const NavigationMenuRoot: React.FC<
+  NavigationMenuPrimitive.NavigationMenuProps
+> = ({ className, ...props }) => (
+  <NavigationMenuPrimitive.Root
+    className={cn("relative z-[1] flex flex-1 justify-center", className)}
+    {...props}
+  />
+)
+
+export const NavigationMenuList: React.FC<
+  NavigationMenuPrimitive.NavigationMenuListProps
+> = ({ className, ...props }) => (
+  <NavigationMenuPrimitive.List
+    className={cn(
+      "group flex flex-1 list-none justify-center rounded-lg border border-slate-300 bg-white p-1 dark:border-slate-700 dark:bg-slate-800",
+      className
+    )}
+    {...props}
+  />
+)
+
+export const NavigationMenuItem = NavigationMenuPrimitive.NavigationMenuItem
+
+const navigationMenuTriggerStyle = cva(
+  "rounded-md py-2 px-3 font-medium leading-none hover:bg-slate-100 focus:shadow-sm focus:ring-2 focus:ring-slate-300 focus:bg-slate-100 dark:focus:bg-slate-700 dark:hover:bg-slate-700 dark:focus:ring-slate-500 outline-none w-full transition-all duration-300"
+)
+
+export const NavigationMenuTrigger: React.FC<
```

Comment:
We need to add the `displayName` for the components.

`NavigationMenuTrigger.displayName = NavigationMenuPrimitive.Trigger.displayName`

Reply:
It's already the same displayName no ?
<img width="1219" alt="image" src="https://user-images.githubusercontent.com/16818589/215133389-14dba6af-e8f4-4a28-8407-f0f8e62a25af.png">